src/apps/Apps/InstalledApps/AppContent.js

```diff
@@ -1,132 +1,132 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { Button, IdentityBadge, breakpoint } from '@aragon/ui'
+import { Button, IdentityBadge, Viewport } from '@aragon/ui'
 import { TextLabel } from '../../../components/TextStyles'
-import Screenshots from '../Screenshots'
 import { GU } from '../../../utils'
+import { MENU_PANEL_WIDTH } from '../../../components/MenuPanel/MenuPanel'
+import Screenshots from '../Screenshots'
+
+// Exclude the width of MenuPanel
+const appBelow = (below, value) =>
```

Comment:
With this change, you want to normalize app responsive behavior to simulate iframe app dimensions no?

Reply:
Yes, experimenting for now but that’s the idea :-)